src/hw/getDeviceInfo.ts

```diff
@@ -37,6 +39,7 @@ export default async function getDeviceInfo(
     throw new DeviceOnDashboardExpected();
   }
 
+  const onboarding = await getOnboardingStatus(transport);
```

Comment:
Why on earth would you call getVersion right before getversion on the next line?
This is completely backwards, sending the apdu twice whenver we want to `getDeviceInfo`